config/initializers/simple_form.rb

```diff
@@ -27,7 +27,7 @@
     b.optional :pattern
     b.optional :min_max
     b.optional :readonly
-    b.use :label, class: 'text-bold'
+    b.use :label, class: 'usa-label'
```

Comment:
I'm getting a bit brave with some of these form default design system migrations, but we're long overdue to rip off this bandaid.

Reply:
:shipit: